### DISCUSSION TITLE
This library is damn helpful for me. Thank you AmrDeveloper

### DISCUSSION THREAD

**Discussion by socartik** [AmrDeveloper/CodeView#25] [Show and tell]
![Screenshot_20220429-154408_InstaLatex.jpg](https://user-images.githubusercontent.com/77521018/165926340-d075f961-df06-46be-9ad2-a2c592f73b4c.jpg)

**Comment by AmrDeveloper** [comment]
Hello @socartik,

You're most welcome, I am happy that you enjoyed it,

Thanks for the screenshot demo and good luck

Amr Hesham :D
